lib/maxmind/geoip2/client.rb

```diff
@@ -245,11 +245,15 @@ def response_for(endpoint, model_class, ip_address)
       end
 
       def make_http_client
+        client_version = Gem::Specification.load('maxmind-geoip2.gemspec').version.version
```

Comment:
I don't think this'll be possible, at least with not some additional work to locate the gemspec file, which I'm not sure is a good idea. You can try running this outside the repo and it'll throw an error, e.g.

```
/home/wstorey/.rubies/ruby-3.2.0/lib/ruby/gems/3.2.0/gems/maxmind-geoip2-1.2.1/lib/maxmind/geoip2/client.rb:248:in `make_http_client': undefined method `version' for nil:NilClass (NoMethodError)

        client_version = Gem::Specification.load('maxmind-geoip2.gemspec').version.version
                                                                          ^^^^^^^^
        from /home/wstorey/.rubies/ruby-3.2.0/lib/ruby/gems/3.2.0/gems/maxmind-geoip2-1.2.1/lib/maxmind/geoip2/client.rb:114:in `block in initialize'
```

From some quick searching, it looks like a convention is to use a `VERSION` constant to hold the version info. We do that in the `minfraud` gem. We could probably do that here.

Reply:
1.1.0 since what we've been doing is keeping the current version set in the repo until we release.

Reply:
How so? The majority will be using a released version and it will be set correctly each release.

Reply:
I think it'd be pretty unusual to go outside of using it via the gem, so I am not worried about trying to capture it. However if you want to add a `-dev` to the version in the repo, that's fine. If so, can you add something to `README.dev.md` about it so we hopefully remember when we do releases?